### PR TITLE
Refactor/metadata model

### PR DESCRIPTION
**Pull request by MirkoFer-622**
Some classes are changed for adapting to refactoring of project.

### CODE REVIEW

File: src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java [hunk 1]
```diff
@@ -42,6 +42,13 @@ public String getToolExtactor() {
    * Returns executable meta data extractor path.
    */
   public String getMetaDataExtractorTool() {
-    return properties.getProperty("exiftool.path");
+    return properties.getProperty("metadata.path");
```
Comment (maryangela) [inline src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java:49]: please review the identations

File: src/main/java/org/fundacionjala/converter/executor/Executor.java [hunk 2]
```diff
@@ -3,46 +3,65 @@
...
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.ExecutionException;
 
 public class Executor {
 
-  private boolean debugOutPut;
+    private boolean debugOutPut;
```
Comment (maryangela) [inline src/main/java/org/fundacionjala/converter/executor/Executor.java]: please change the name to debugOutput

File: src/main/java/org/fundacionjala/converter/model/command/AudioModel.java [hunk 3]
```diff
@@ -1,4 +1,6 @@
 package org.fundacionjala.converter.model.command;
+import org.fundacionjala.converter.model.parameter.ModelParameter;
+
```
Comment (maryangela) [inline src/main/java/org/fundacionjala/converter/model/command/AudioModel.java]: please delete this blank line

**Comment by julia-jpg** [comment]
solve conflicts

**Review by maryangela — COMMENTED** [review]
please solve the conflicts

**Comment by julia-jpg** [comment]
image
could you delete the files
- MetadataParam.java
- Param.java

File: src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java (hunk 1 above)
Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java:49]: I solve it, could you review please?

File: src/main/java/org/fundacionjala/converter/model/command/AudioModel.java (hunk 3 above)
Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/model/command/AudioModel.java]: I solve, could you review it please?

File: src/main/java/org/fundacionjala/converter/model/command/ImageModel.java [hunk 4]
```diff
@@ -97,7 +98,7 @@ public String grayScale(final MultipartFile file, final String newName, final St
      * @return list of commands
      */
     @Override
-    public List<String> createCommand() {
+    public List<List<String>> createCommand(final ModelParameter modelParameter) {
         return null;
     }
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/model/command/ImageModel.java]: it is ok to return just null?

File: src/main/java/org/fundacionjala/converter/model/command/MetadataModel.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+            case "j":
+                return ".json";
+            case "h":
+                return ".html";
+            case "t":
+                return ".txt";
+            case "T":
+                return ".csv";
+            default:
+                return ".XMP";
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/model/command/MetadataModel.java:50]: try to use enums on extensions please

File: src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java [hunk 6]
```diff
@@ -5,43 +5,50 @@
...
+    /**
+    * Loads the application.properties configurations.
+    */
+    public void loadConfiguration() {
+        properties = new Properties();
+        InputStream inputPath = ConfigPath.class.getClassLoader().getResourceAsStream("application.properties");
+        try {
+            properties.load(inputPath);
+        } catch (IOException e) {
+          e.printStackTrace();
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java]: the indentation please

File: src/main/java/org/fundacionjala/converter/model/command/ImageModel.java (hunk 4 above)
Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/model/command/ImageModel.java]: Yes, it is not my work domain, I just am keeping it for compilation.

File: src/main/java/org/fundacionjala/converter/model/command/MetadataModel.java (hunk 5 above)
Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/model/command/MetadataModel.java:50]: For this time, Team did not consider use of enums for all project, so this enhancement will be present in the next sprint.

File: src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java (hunk 6 above)
Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/model/configPath/ConfigPath.java]: I fixed it, could you review please?

**Review by samuelloza — APPROVED** [review]

**Comment by samuelloza** [comment]
The MetadataParameter.java have conflic, when try merge with dev
image

**Review by Laura4lilavati — APPROVED** [review]

**Review by julia-jpg — APPROVED** [review]

**Pull request merged**